.github/workflows/jekyll.yml

```diff
@@ -27,37 +28,44 @@ concurrency:
   cancel-in-progress: false
 
 jobs:
-  # Build job
   build:
     runs-on: ubuntu-latest
+
     steps:
       - name: Checkout
         uses: actions/checkout@v4
+
       - name: Setup Ruby
         uses: ruby/setup-ruby@f321cf5a4d1533575411f8752cf25b86478b0442 # v1.193.0
         with:
           ruby-version: '3.1' # Not needed with a .ruby-version file
           bundler-cache: true # runs 'bundle install' and caches installed gems automatically
           cache-version: 0 # Increment this number if you need to re-download cached gems
+
       - name: Setup Pages
         id: pages
         uses: actions/configure-pages@v5
```

Comment:
I guess `actions/configure-pages` doesn't hurt from `actions/deploy-pages` being cancelled. The "However, do NOT cancel in-progress runs as we want to allow these production deployments to complete." on line 25 threw me off 😅